charts/events/values.yaml

```diff
@@ -0,0 +1,36 @@
+argo-events:
+  enabled: true
+  controller:
+    replicas: 1
```

Comment:
This should probably be greater than one, a PDB of 1 doesn't really work with one replica

Reply:
increased to 3

Reply:
Actually, events with replicas > 1 seems to cause "Failed to load global configuration file" on the controller pod. Reverted back to single replica for now. We can look into this again as part of HA for events triggering

Reply:
Sounds reasonable, can you stick an issue in to make events HA?

Reply:
see #14